src/checkout.c

```diff
@@ -1875,7 +1928,34 @@ static int checkout_deferred_remove(git_repository *repo, const char *path)
 #endif
 }
 
-static int checkout_create_the_new(
+static int checkout_create_perform(
+	checkout_data *data,
+	unsigned int action,
+	git_diff_delta *delta)
+{
+	int error = 0;
+
+	if (action & CHECKOUT_ACTION__DEFER_REMOVE) {
+		/* this had a blocker directory that should only be removed iff
+		 * all of the contents of the directory were safely removed
+		 */
+		if ((error = checkout_deferred_remove(data->repo, delta->old_file.path)) < 0)
```

Comment:
Just for kicks, the body of this function is `#ifdef`ed out to do nothing 😂.

Reply:
:joy:

Reply:
That block is 6 years old. I'm assuming it's not going to change anytime soon. I think I'll remove that then.